BasePolicies/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+in the Node.js ecosystem. Projects are encouraged to adopt this whether they
+are hosted in the Node.js or not.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and beuracracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+This document is based on much prior art in the Node.js community, io.js,
+and the Node.js project.
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
+* A **TC** is a group of committers representing the required technical expertise to 
+resolve rare disputes.
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in doubt, log an issue, 
+any additional policies about what to include will be provided in the responses.
+
+Committers may direct you to another repository, ask for additional clarifications, and
+add appropriate metadata before the issue addressed.
+
+Please be courteous, respectful, and follow the project's Code of Conduct.
+
+# Contributions
+
+Any change to resources in this repository must be pull requests. This applies to all change
+to documentation, code, binary files, etc. Even long term committers and TC members must be
+pull requests.
+
+No pull request can be merged by the person that created it.
+
+For non-trivial contributions pull requests should sit for at least 36 hours to ensure that
+contributors in other timezones have time to review.
+
+The default for each contribution is that it is accepted once no commiter has an objection.
+There is no additional "sign off" process for contributions to land. Once all issues brought
+by committers is addressed it can be landed by any committer. The majority of contributions
+should land this way without escalation to the TC.
+
+If a contribution is controversial and committers cannot agree about how to get it to land
+or if it should land then it should be escalated to the TC. TC members should regularly
+discuss pending contributions in order to find a resolution.
+
+# Becoming a Committer
+
+All contributors who send a non-trivial contribution should be on-boarded in a timely manner,
+and added as a committer, and be given write access to the repository.
+
+Committers are expected to follow this policy and continue to send pull requests, go through
+proper review, and have other committers merge their pull requests.
+
+# TC Process
+
+The TC uses a "consensus seeking" process for issues that are esclated to the TC. 
+The group tries to find a resolution that has no open objections among TC members.
+If a resolution cannot be reached that has no objections then a majority wins vote
```

Comment:
changing to "consensus" rather than "resolution." the problem with unanimous is that it implies that we need to force TC members who don't have an opinion to vote one way or another.